omod/pom.xml

```diff
@@ -67,6 +79,13 @@
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>webservices.rest-omod-common</artifactId>
 		</dependency>
+       	
+		<dependency>
+	        <groupId>org.javassist</groupId>
```

Comment:
yes, I have built all profiles/versions and they run fine with this version of javaassist